usefulUtilities/hiFiCalendar/clientScripts/meetingRoomSetup_app.js

```diff
@@ -0,0 +1,161 @@
+//  meetingRoomSetup_app.js
+//
+//  Created by Mark Brosche on 4-2-2019
+//  Handed off to Milad Nazeri on 5-15-2019
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function() {
+    var CONFIG = Script.require("../calendarConfig.json?" + Date.now());
+    var CHANNEL = "HiFi.Google.Calendar";
+    var DOMAIN = "hq";
+    var MS_TO_SEC = 1000;
+    var MIN_PER_HR = 60;
+    var TOKEN_SERVER_ID = CONFIG.TOKEN_SERVER_ID;
```

Comment:
does this still need to be set in the config file if we're getting info about it from the messages mixer?

Reply:
Not really, removed.